### PR TITLE
🚨 fix markdownlint issues, speak up link, OpenJSF link

### PR DESCRIPTION
**Pull request by ctcpip**

### CODE REVIEW

**Comment by ctcpip** [comment]
speak up link was changed per https://github.com/django/djangoproject.com/issues/712

File: index.md [hunk 1]
```diff
@@ -113,11 +111,11 @@ Responses will be determined by the Committee on the basis of the information ga
 
 Such actions taken by the Committee will be reported as part of the TC39 report to the ExeCom.
 
-The Committee will never publicly discuss the issue; all public statements, if needed, will be made by the TC39 Chair and/or the Ecma Secretariat.
+The Committee will never publicly discuss the issue; all public statements, if needed, will be made by the TC39 Chair and/or the Ecma Secretariat.
```
[the remark this answers is not in the document]
Comment (ctcpip) [reply index.md:114]: > The character U+037e ";" is not a basic ASCII character.

`';' !== ';'`

[the remark this answers is not in the document]
Comment (ljharb) [reply index.md]: hm, when i copy-paste it they end up as the same character - i think maybe my mac auto-normalizes them all to a regular semicolon

**Review by ljharb — APPROVED** [review]

File: index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ctcpip) [reply index.md]: weird.. my mac does not.  if I copy that exact code above, I get true.  it's the Greek question mark character 🤷‍♀️ 


Screen Shot 2022-04-29 at 3 08 10 PM

[the remark this answers is not in the document]
Comment (ljharb) [reply index.md]: Maybe it's Safari then; that looks like it's something else?

[the remark this answers is not in the document]
Comment (ctcpip) [reply index.md]: that's Chrome.  yep, sure enough if I paste it in Safari, it's false.  AND, if I copy it from Safari and I entered it in, it's false back in Chrome, so Safari is changing it.  wild stuff

**Pull request merged**